myapp/src/day2/Question14/Tasks.jsx

```diff
@@ -0,0 +1,51 @@
+import { useCallback, useState } from 'react'
+import Task from './Task'
```

Comment:
all tasks are getting re-rendered when one task is completed

Reply:
Yes, because setTasks triggers a re-render of the Tasks component, which in turn re-renders its child components since they are mapped inside the div